manager/controllers/utils.go

```diff
@@ -47,6 +47,17 @@ func LogAndRespNotFound(c *gin.Context, err error, respMsg string) {
 	c.AbortWithStatusJSON(http.StatusNotFound, utils.ErrorResponse{Error: respMsg})
 }
 
+func createOrderFields(fieldExprs database.AttrMap, order string, enteredField string) string {
+	var orderFields string
+	if enteredField == "os_name" {
+		orderFields = fmt.Sprintf("%s %s NULLS LAST,%s %s NULLS LAST,%s %s NULLS LAST",
```

Comment:
To ensure the same behaviour we need to use `NULLS LAST` and `NULLS FIRST` according to `order` string variable. Btw. it would be good to use null value in the test to verify that. Actually we can include `NULLS LAST/FIRST` into the `order` input variable, right? So we will use order = `DESC NULLS LAST` or order = `ASC NULLS FIRST`.